test/testInlineStylesheets/index.html

```diff
@@ -0,0 +1,3 @@
+<link href="/author.html" rel="author">
+<link href="myStyle.css" rel="stylesheet">
+<link href="nested/myNestedStylesheet.css" rel="stylesheet" />
```

Comment:
This is good for checking the replacement works. But I also want to make sure the non-trivial regex is working.

```
Link tags without `rel="stylesheet"` should not be replaced.

Not a stylesheet
<link href="/author.html" rel="author">

Missing rel attribute
<link href="myStyle.css">

Incorrect syntax
<linkrel="stylesheet" href="myStyle.css">


Link tags with `rel="stylesheet"` should be replaced.

<link href="myStyle.css" rel="stylesheet">

Space between attributes is optional
<link rel="stylesheet"href="myStyle.css">

<link href="nested/myNestedStylesheet.css" rel="stylesheet" />
```

Reply:
Ah, I see now. I've updated the test, seems to work fine!